Dashboard-locale/locales-whitelabel.json

```diff
@@ -1,127 +1,115 @@
 {
   "whitelabel": {
-    "login-sign-in": {
+    "header-subtitle-new": {
+      "dutch": "Uw {0}-portemonnee met één klik",
+      "english": "Your {0} wallet with one click",
+      "french": "Votre portefeuille {0} en un clic",
+      "german": "Ihre {0}-Brieftasche mit einem Klick",
+      "japanese": "ワンクリックで {0} ウォレット",
+      "korean": "한 번의 클릭으로 {0} 지갑",
+      "mandarin": "一键点击您的 {0} 钱包",
+      "portuguese": "Sua carteira {0} com um clique",
+      "spanish": "Su billetera {0} con un solo clic",
+      "turkish": "Tek tıklama ile {0} cüzdanınız"
+    },
+    "header-title": {
+      "dutch": "Aanmelden",
       "english": "Sign in",
-      "german": "anmelden",
+      "french": "Se connecter",
+      "german": "Einloggen",
       "japanese": "ログイン",
       "korean": "로그인",
-      "mandarin": "登入",
-      "spanish": "Iniciar sesión",
+      "mandarin": "登录",
       "portuguese": "Entrar",
```

Comment:
this will fail on parsing the json.. remove extra comma on each last child

Reply:
nice let me modify